orderer/blockattestation.proto

```diff
@@ -16,14 +16,18 @@ message BlockAttestation {
        common.BlockMetadata metadata = 2;
 }
 
-message BlockAttestationResponse {
-    oneof Type {
+message BlockAttestationsRequest {
+    common.Envelope envelope = 1;
+}
+
+message BlockAttestationsResponse {
+    oneof type {
         common.Status status = 1;
         BlockAttestation block_attestation = 2;
     }
 }
 
-service BlockAttestations {
+service BlockAttestationsService {
     // BlockAttestations receives an Envelope of type DELIVER_SEEK_INFO , then sends back a stream of BlockAttestations.
-    rpc BlockAttestations(common.Envelope) returns (stream BlockAttestationResponse);
+    rpc BlockAttestations(BlockAttestationsRequest) returns (stream BlockAttestationsResponse);
```

Comment:
whether the linter likes it or not, we have `Envelope`s everywhere in our codebase, so calling it something else would just be more confusing to programmers.

Reply:
This is the point raised by @jt-nti , 

```
Having a service specific request message is probably the most important warning to fix. The request can include the required common.Envelope but could then evolve independently of the envelope in the future if required.

```
It makes sense if we want to extend the parameter independently then wrapping the common.Envelope into service specific request message should be good idea.

Reply:
This is the point raised by @jt-nti , 

```
Having a service specific request message is probably the most important warning to fix. The request can include the required common.Envelope but could then evolve independently of the envelope in the future if required.

```
It makes sense if we want to extend the parameter independently then wrapping the common.Envelope into service specific request message should be good idea.

Reply:
As you are aware (since you implemented it), the service is implemented by the common deliver server infrastructure, so it it's unlikely we will need to change the parameter sent by the client.